robot_smach_states/src/robot_smach_states/util/designators/utility.py

```diff
@@ -80,6 +80,9 @@ def resolve(self):
     def __repr__(self):
         return "LockingDesignator({})".format(str(self.to_be_locked.name))
 
+    def _get_name(self):
```

Comment:
When the designator is __repr__-ed or __str__-ed, i.e. when you want to print it or it's name. Then you can look them up in your code more easy. 
The names are also used in the designator visualisation